.travis.yml

```
@@ -33,21 +34,24 @@ before_script:
   - docker pull $DOCKER_PREFIX/xeon-$(echo $buildOS | tr -d - | tr -d .)-$usage-$imageName:build || true
   - docker pull $DOCKER_PREFIX/xeon-$(echo $buildOS | tr -d - | tr -d .)-$usage-$imageName:latest || true
 script:
-  - &basescript |
-    mkdir build && cd build && cmake -DBUILD_MP3LAME=OFF -DBUILD_FDKAAC=OFF -DDOCKER_PREFIX=${DOCKER_PREFIX} ..
-    cd ${CPU}/${buildOS}/${usage}/${imageName}
-  - make
-  - travis_retry ctest -V
-after_success: |
+  - mkdir build && cd build && cmake -DBUILD_MP3LAME=OFF -DBUILD_FDKAAC=OFF -DDOCKER_PREFIX=${DOCKER_PREFIX} ..
+  - travis_wait 180 ${TRAVIS_BUILD_DIR}/script/travis_build.sh ${CPU}/${buildOS}/${usage}/${imageName}
+  - tail --lines=5000 ${TRAVIS_BUILD_DIR}/build/travis.log
```

```suggestion
  - travis_wait 180 "${TRAVIS_BUILD_DIR}/script/travis_build.sh" "${CPU}/${buildOS}/${usage}/${imageName}"
  - tail --lines=5000" ${TRAVIS_BUILD_DIR}/build/travis.log"
```

you may want to quote these just in case to prevent any more expansion past variable expansions